### ISSUE TITLE
Readings for Validator Privacy

### ISSUE THREAD

**Issue by chloethedev** [hashcloak/research#3]
Chapters 1, 2, 6 of Threat Modeling

- [x] All ethresearch post related to validator privacy

- [x] https://moodle.ufsc.br/pluginfile.php/2377555/mod_resource/content/2/Threat%20Modeling.pdf

- [x] https://www.esat.kuleuven.be/cosic/publications/thesis-187.pdf

- [x] https://devavrat.mit.edu/wp-content/uploads/2017/10/Rumors-in-a-network-whos-the-culprit.pdf

- [x] https://petsymposium.org/2019/files/hotpets/proposals/coordination-helps-anonymity-new.pdf

- [x] https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-594.pdf

- [x] https://www.freehaven.net/anonbib/cache/george-thesis.pdf

- [x] http://asirap.net/work/msthesis.pdf

- [x] https://eth.wiki/concepts/whisper/whisper

- [x] https://pdfs.semanticscholar.org/46a3/a71ea37d7024c8309dbb1412c84ffbc7a942.pdf

- [x] https://www.cl.cam.ac.uk/~rja14/Papers/redblue.pdf

- [x] https://www.onion-router.net/Publications/challenges.pdf
- [x] https://notes.ethereum.org/@adiasg/weak-subjectvity-eth2
- [x] https://eprint.iacr.org/2020/025
- [x] https://petsymposium.org/2019/files/papers/issue2/popets-2019-0022.pdf
- [x] https://arxiv.org/pdf/1710.10237.pdf
- [x] https://www.freehaven.net/anonbib/cache/ih05-danezisclulow.pdf
- [x] https://www.freehaven.net/doc/fc03/econymics.pdf
- [x] https://www.freehaven.net/anonbib/cache/morphmix:wpes2002.pdf
- [x] https://dl.acm.org/doi/pdf/10.5555/1274531.1274553?download=true
- [x] https://www.freehaven.net/anonbib/cache/topology-pet2010.pdf
- [x] https://www-users.cs.umn.edu/~hoppernj/hashing_it_out.pdf
- [x] https://github.com/hashcloak/phantom / video: https://www.youtube.com/watch?v=IAJn3Ag3imY
- [ ] https://github.com/hashcloak/BitBlinder

**Comment by chloethedev** [comment]
Link for Hack MD: https://hackmd.io/BBBg1YxHRU6GEhfbkzYIbg

Working on "All ethresear.ch posts related to validator privacy"

**Comment by chloethedev** [comment]
Went through the obvious posts on ethresear.ch that centered around "Validator Privacy": https://hackmd.io/BBBg1YxHRU6GEhfbkzYIbg

Started on the first paper here: https://hackmd.io/TPh4pjikQE-_RNqMsqBRRQ

**Comment by chloethedev** [comment]
Starting with SSLE paper. Link @ https://hackmd.io/BNBb2U39Rx6E-iRwVGox6A

**Comment by chloethedev** [comment]
Moving on to https://hackmd.io/k08e0MMYRIuRwHHtCWFOXQ.

**Comment by chloethedev** [comment]
Started working on "Rumors in a Network: Who's the culprit?" Notes are available at: https://hackmd.io/gM7FXo4bQi2KVS9C_yvtbA?edit

**Comment by chloethedev** [comment]
https://www.freehaven.net/anonbib/cache/george-thesis.pdf seems to be the same as https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-594.pdf. Could be wrong, remember to ask.

**Comment by Mikerah** [comment]
@chloethedev Nope, they are different. One is Danezis's master's thesis and the other his PhD thesis. There's overlapping content though.

**Comment by Mikerah** [comment]
Actually, both seem to be his PhD thesis but one is more of a technical report. There's overlapping content in both. Just read the whichever one is easiest.

**Comment by chloethedev** [comment]
> Actually, both seem to be his PhD thesis but one is more of a technical report. There's overlapping content in both. Just read the whichever one is easiest.

One of them, https://www.freehaven.net/anonbib/cache/george-thesis.pdf, did not show up well on my browser so I went through the technical report which I noted in the notes. Thank you for the response.

**Comment by chloethedev** [comment]
Finished with the Whisper paper at https://hackmd.io/0NyMktZyQx-sr8w-uxo25w?view

**Comment by chloethedev** [comment]
Complete: https://hackmd.io/H3eDeCedQwyf9LWssDK_dw?view

**Comment by chloethedev** [comment]
Starting: https://hackmd.io/LNdM-ijfQ1i9Z9Zl38PHgg?view

**Comment by chloethedev** [comment]
Completed https://hackmd.io/hOyMGL33SNWbQC8AJRkicQ?view Setting up https://hackmd.io/89aDiXZ8RXODmzeR756sQg?view for Monday.

**Comment by chloethedev** [comment]
1. Validator Privacy Notes from ETHresear.ch:
https://hackmd.io/BBBg1YxHRU6GEhfbkzYIbg
2. Summary of Designing for Security - Adam Shostack CH - 1,2,6:
https://hackmd.io/TPh4pjikQE-_RNqMsqBRRQ
3. Design and analysis methods for privacy technologies - Carmela Troncoso 2011: https://hackmd.io/k08e0MMYRIuRwHHtCWFOXQ
4. Rumors in a Network: Who’s the culprit? - 2011: 
https://hackmd.io/gM7FXo4bQi2KVS9C_yvtbA
5. Designing and attacking anonymous communication systems - George Danezis 2004: https://hackmd.io/lAN4_-nCTyWzftOb05fbYw
6. Not all is lost for anonymity – but quite a lot is. - 2019:
https://hackmd.io/QTsKIjBeRbGTwYEtSYV-oQ?view
7. Challenges in deploying low-latency anonymity (DRAFT):
https://hackmd.io/hOyMGL33SNWbQC8AJRkicQ
8. Weak Subjectivity in Eth2.0 - 2020:
https://hackmd.io/nk4zhjcLSWO6URS9z5Ux_w
9. The Economics of Censorship Resistance:
https://hackmd.io/LNdM-ijfQ1i9Z9Zl38PHgg
10. Whisper: 
https://hackmd.io/0NyMktZyQx-sr8w-uxo25w
11. An Excess-Based Economic Model for Resource Allocation in Peer-to-Peer Networks:
https://hackmd.io/H3eDeCedQwyf9LWssDK_dw
12. BYZANTINE ATTACKS ON ANONYMITY SYSTEMS - 2005:
https://hackmd.io/mE_-BJ2WQf2JLrn73gbeJA
13. Secret Leader Election:
https://hackmd.io/BNBb2U39Rx6E-iRwVGox6A
14. On Privacy Notions in Anonymous Communication - 2019:
https://hackmd.io/89aDiXZ8RXODmzeR756sQg 
15. Compulsion Resistant Anonymous Communications:
https://hackmd.io/nexCkSZGREiNzR0xPCumYA?view
16. On the Economics of Anonymity:
https://hackmd.io/YKg4qh_0RYiO65OKk5sUBw?view
17. Introducing MorphMix: Peer-to-Peer based Anonymous Internet Usage with Collusion Detection:
https://hackmd.io/2_RlHuUGSjyKYLJdxmyTfg?view
18. Design Principles for Low Latency Anonymous Network Systems Secure against Timing Attacks:
https://hackmd.io/yYvAhne_RFSkoDIbRFQWmA?view
19. Impact of Network Topology on Anonymity and Overhead in Low-Latency Anonymity Networks:
https://hackmd.io/LzdR0zH8TAm-au3lGkhqJQ?view
20. Hashing it out in public:
https://hackmd.io/ELqpWt-qTvWgTX-PZ9dMbg?view
21. Video: DEFCON 16: Generic, Decentralized, Unstoppable Anonymity: The Phantom Protocol:
https://hackmd.io/bfijThaDSjWiSYnM44d5SQ?view
